src/features/pages/404-error/content/data.ts

```diff
@@ -0,0 +1,15 @@
+import { ButtonLink, InfoItem, ImageItem } from './types'
+import NotAvailableImage from 'images/svg/no_page_found.svg'
+
+export const not_available_image: ImageItem = {
+    src: NotAvailableImage,
+    alt: 'Page not found',
```

Comment:
alt images should be translatable, please update this and translate this as well. thanks :pray:

Reply:
okay thanks @mohsen-deriv